Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -2246,13 +2246,14 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Give SG a rev history with a gap",
     }
 }
 
-// To run this test, the sync function for Tulips collection must be,
-// "tulips":{"sync":"function(doc,olddoc){if(doc.cbl_445==\"bad\")throw({\"forbidden\":\"read_only\"});channel(doc.channels)}"}
-TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Use isRevRejected to Resolve Conflict", "[.SyncServerCollection]") {
+// This test uses collection, "cbl443", which has the following sync function:
+// "cbl445":{"sync":"function(doc,olddoc){if(doc.cbl_445==\"bad\")throw({\"forbidden\":\"read_only\"});channel(doc.channels)}"}
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Use isRevRejected to Resolve Conflict", "[.xSyncServerCollection]") {
```

Comment:
Can you update here as well?

https://github.com/couchbase/couchbase-lite-core/blob/2f5430f6a7b6644b7b21f557e7f1f33282270aff/Replicator/tests/ReplicatorCollectionSGTest.cc#L20-L63

Maybe use another flower name such as "marigold" for consistency.

Reply:
Why another flower? What would be another flower different from Tulips which I used originally